### PR TITLE
Streaming: Fix an issue with the time series panel and streaming data source when scrolling back from being out of view

### LINKED ISSUE #31368: Time Series panel is refreshing and loosing previous values when edit or scrolling dashboard
**What happened**:

The Time Series panel behaves differently than the Graph panel. When scrolling the dashboard up and down and the Time Series panel is not visible it's refreshing and losing collected values. Graph panel keeps all the values.
Also, when clicking on panel edit and don't change anything, return back to the dashboard, panel is refreshing.

Play.grafana.com working similar because provides historical data filling the whole time-range. In case of Redis Data source, there is no historical values.

**What you expected to happen**:

Be similar to the Graph panel and keeps values. Or this behaviour is expected?

**How to reproduce it (as minimally and precisely as possible)**:

1.  Use the latest version of Redis Data Source or other Streaming datasource
```
docker run -p 3000:3000 ghcr.io/redisgrafana/redis-app:latest
```

2. Import Dashboard with Time-series panels or create new Time-Series panel
```
https://raw.githubusercontent.com/RedisGrafana/grafana-redis-datasource/master/provisioning/dashboards/data-types-7.4.json
```

3. Scroll dashboard up and down or edit the panel.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.4.2
- Data source type & version: Redis Data Source 1.1.0
- OS Grafana is installed on: Docker
- User OS & Browser:
- Grafana plugins: Redis Data Source
- Others:

### PR DESCRIPTION
**Pull request by torkelo**
Fixes #31368

### CODE REVIEW

File: public/app/features/dashboard/dashgrid/PanelChrome.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ export class PanelChrome extends Component<Props, State> {
   // The next is outside a react synthetic event so setState is not batched
   // So in this context we can only do a single call to setState
   onDataUpdate(data: PanelData) {
-    if (!this.props.isInView) {
+    if (!this.props.isInView && data.state !== LoadingState.Streaming) {
```
Comment (torkelo) [inline public/app/features/dashboard/dashgrid/PanelChrome.tsx:138]: wait this is not correct, need to always return, updating

**Review by ryantxu — APPROVED** [review]
hymmm -- this may fix it, but likely a real error with state/revision tracking and updates in GraphNG

**Comment by torkelo** [comment]
This fixed the issue for me, without this fix there is a refresh when you scroll back

**Comment by leeoniya** [comment]
> hymmm -- this may fix it, but likely a real error with state/revision tracking and updates in GraphNG

bug report was for 7.4.2 (before graphng refactor), does this still affect master? (sry AFK)

**Comment by torkelo** [comment]
This bug affected all react panels . Had nothing to do with GraphNG

**Pull request merged**